jetty-server/src/main/java/org/eclipse/jetty/server/MultiPartFormInputStream.java

```diff
@@ -185,12 +186,17 @@ protected void write(byte[] bytes, int offset, int length) throws IOException
         @Override
         public void write(String fileName) throws IOException
         {
+            Path p = Path.of(fileName);
+            
```

Comment:
You have the same isAbsolute check in both branches below so perhaps do that here.
```suggestion
            Path p = Path.of(fileName);
            p = p.isAbsolute() ? p : _tmpDir.resolve(p);
```

Reply:
Note that while I know resolve also handles absolute addresses, I do kind of like seeing it explicit in this code, so you can understand it without the need to have deep knowledge of how resolve behaves.   Maybe have a comment to that effect?